CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog]
 
 ## [Unreleased]
 
+- Show date of birth in a better place and format for claim checking
```

Comment:
I'd argue that any format other than ISO8601 isn't 'better' :trollface:

Reply:
I'm not getting into your date fetish! 😄